src/condor_ce_scitoken_exchange

```diff
@@ -0,0 +1,142 @@
+#!/usr/bin/python3
+
+import argparse
+import base64
+import binascii
+import json
+import os
+import subprocess
+import sys
+import time
+
+os.environ.setdefault("CONDOR_CONFIG", "/etc/condor-ce/condor_config")
+
+import htcondor
+
+
+DEFAULT_COLLECTOR_PORT="9619"
+
+def parse_args():
+    parser = argparse.ArgumentParser(description="Exchange a SciToken for a HTCondor token at a remote CE")
+    parser.add_argument("-pool", help="Remote collector to query", metavar="coll",
+        default="collector.opensciencegrid.org")
```

Comment:
I was thinking about this a bit more and this means that for this to work consistently, we need the collector1 <-> collector2 ad sharing feature implemented in HTCondor, then the HTCondor-CE central collector config.

Reply:
I'm not sure I follow here -- this should work with either collector just as well as it does today.

Reply:
Why do we have the OSG collector as the default collector? I thought this PR was not supposed to be OSG-specific.

If you don't want to make everyone specify `-pool` then there should be a condor config knob for the default collector.

Reply:
> I'm not sure I follow here -- this should work with either collector just as well as it does today.

If a CE has only successfully advertised to one of the collectors and the query gets DNS round-robined to the other collector, the command will fail. Admittedly, I think we've only seen this scenario come up once so I don't think it should hold up the PR but I thought it was worth highlighting.

> Why do we have the OSG collector as the default collector? I thought this PR was not supposed to be OSG-specific.
> 
> If you don't want to make everyone specify `-pool` then there should be a condor config knob for the default collector.

We also have this problem in [condor_ce_info_status](https://github.com/htcondor/htcondor-ce/blob/master/src/condor_ce_info_status#L3) and I've been trying to think of a good default. A config knob (`CONDORCE_CENTRAL_COLLECTOR`?) is a good idea but I'm not sure if we should leave it undefined and have the tools that depend on it fail with a message or if defaulting to the OSG collector would be sufficient. Regardless, I think we should address this in a subsequent PR.

Reply:
OSG's collectors do not run in DNS round-robin.

Reply:
Ok - if I'm reading this right, it's an item to note but not an action item for this PR.  Resolving.